content/en/docs/reference/ports-and-protocols.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ports and Protocols
+content_type: concept
+weight: 50
+---
+
+When running Kubernetes in an environment with strict network boundaries, such 
+as on-premises datacenter with physical network firewalls or Virtual 
+Networks in Public Cloud, it is useful to be aware of the ports and protocols 
+used by Kubernetes components
+
+## Control-plane node(s)
```

Comment:
```suggestion
## Control plane
```
These ports need to be open even if you run the control plane directly (not within a kubelet).